content/v1.0.0/index.md

```diff
@@ -0,0 +1,200 @@
+  section of the codebase surrounded by parenthesis, e.g., `fix(parser):`
+1. A description MUST immediately follow the space after the type/scope prefix.
+The description is a short summary of the code changes, e.g., _fix: array parsing issue when multiple spaces were contained in string._
+1. A longer commit body MAY be provided after the short description, providing additional contextual information about the code changes. The body MUST begin one blank line after the description.
+1. A footer of one or more lines MAY be provided one blank line after the body. The footer MUST contain meta-information
+about the commit, e.g., related pull-requests, reviewers, breaking changes, with one piece of meta-information
+per-line.
+1. Breaking changes MUST be indicated in the type/scope prefix of a commit, or at the beginning of its optional body or footer section.
+1. If included in the body or footer, a breaking change MUST consist of the uppercase text BREAKING CHANGE, followed by a colon, space, and description, e.g.,
+_BREAKING CHANGE: environment variables now take precedence over config files._
+1. If included in the type/scope prefix, breaking changes MUST be indicated by a
+`!` immediately before the `:`. If `!` is used, `BREAKING CHANGE:` MAY be ommitted from the
+body or footer section, and the commit description SHALL be used to describe the breaking change.
+1. Types other than `feat` and `fix` MAY be used in your commit messages.
+1. The units of information that make up conventional commits MUST NOT be treated as case sensitive by implementors, with the exception of BREAKING CHANGE which MUST be uppercase.
+
+## Why Use Conventional Commits
+
+* Automatically generating CHANGELOGs.
+* Automatically determining a semantic version bump (based on the types of commits landed).
+* Communicating the nature of changes to teammates, the public, and other stakeholders.
+* Triggering build and publish processes.
+* Making it easier for people to contribute to your projects, by allowing them to explore
+  a more structured commit history.
+
+## FAQ
+
+### How should I deal with commit messages in the initial development phase?
+
+We recommend that you proceed as if you've already released the product. Typically *somebody*, even if it's your fellow software developers, is using your software. They'll want to know what's fixed, what breaks etc.
+
+### Are the types in the commit title uppercase or lowercase?
+
+Any casing may be used, but it's best to be consistent.
+
+### What do I do if the commit conforms to more than one of the commit types?
+
+Go back and make multiple commits whenever possible. Part of the benefit of Conventional Commits is its ability to drive us to make more organized commits and PRs.
+
+### Doesn’t this discourage rapid development and fast iteration?
+
+It discourages moving fast in a disorganized way. It helps you be able to move fast long term across multiple projects with varied contributors.
+
```

Comment:
I think we should add an FAQ section that addresses the confusion discussed in https://github.com/conventional-commits/conventionalcommits.org/issues/170

At this time, I don't want to introduce more complexity into the specification, with regards to differentiating body from footer, but I would like to address the conversation that's happening.

_This is not to say we might not, in future iterations of the specification make some changes to our rules around footers. But I don't want to cross this path now._

Reply:
Yeah, I think this is something we need to address before tagging it as 1.0.0